app/models/concerns/moderated.rb

```diff
@@ -117,6 +118,9 @@ def validate_additional_changeset(changes)
   def after_change_accept(change)
   end
 
+  def on_change_suggested
```

Comment:
Should this throw an error if it's not overwritten in the class that uses this mixin?

Reply:
Not in this case, as it will get called every time, but if the implementing class doesn't want to use the callback then a no-op is just fine.